test/rpc_test.py

```diff
@@ -261,53 +264,29 @@ def test_duplicate_name(self):
 
     @dist_init(setup_model_parallel=False)
     def test_reinit(self):
-        dist.init_process_group(
-            backend=dist.Backend.GLOO,
-            init_method=self.init_method,
-            rank=self.rank,
-            world_size=self.world_size,
-        )
         rpc.init_model_parallel(
             self_name="worker{}".format(self.rank),
-            backend=TEST_CONFIG.rpc_backend,
+            backend=rpc.backend_registry.BackendType[TEST_CONFIG.rpc_backend_name],
             init_method=self.init_method,
             self_rank=self.rank,
             worker_name_to_id=self.worker_name_to_id,
         )
         with self.assertRaisesRegex(RuntimeError, "is already initialized"):
             rpc.init_model_parallel(
                 self_name="worker{}".format(self.rank),
-                backend=TEST_CONFIG.rpc_backend,
+                backend=rpc.backend_registry.BackendType[TEST_CONFIG.rpc_backend_name],
                 init_method=self.init_method,
                 self_rank=self.rank,
                 worker_name_to_id=self.worker_name_to_id,
             )
         rpc.join_rpc()
 
-    @dist_init(setup_model_parallel=False)
-    def test_init_invalid_backend(self):
```

Comment:
why removing this test?

Reply:
Because we are changing to Python Enum class. 

`backend_name` string got tested by Enum class internals, which is a Python standard library thing. No need to test Python std library.